### PR TITLE
Enabled changing of themes by clicking on the theme icons

### PR DESCRIPTION
**Pull request by Mystic-Slice**
Resolves:
https://github.com/musescore/MuseScore/issues/7827
https://github.com/musescore/MuseScore/issues/7829

Added Mouse Area on the Theme icons to register mouse clicks. Added a border popup while hovering over the icons to let user know that the icon can be clicked. The border popup color  is set to the respective accent colors.

Changed the displayed text on the reset preferences button

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
the merge commit should not be there

**Comment by RomanPudashkin** [comment]
Please, don't take any bugs that are assigned to someone. I have fixed this issue in my local branch so we made the same work twice

**Comment by Mystic-Slice** [comment]
@RomanPudashkin I am sorry. I did not see that.

File: src/appshell/qml/Preferences/internal/ThemesSection.qml [hunk 1]
```diff
@@ -42,6 +42,21 @@ Column {
                 fontPrimaryColor: modelData.fontPrimaryColor
                 buttonColor: modelData.buttonColor
                 accentColor: modelData.accentColor
+                MouseArea{
+                    anchors.fill: parent
+                    onClicked: {
+                        root.themeChangeRequested(model.index)
+                    }
+                    hoverEnabled: true
+                    onEntered: {
```
Comment (RomanPudashkin) [inline src/appshell/qml/Preferences/internal/ThemesSection.qml:51]: Please, add empty lines before all slots (onClicked, onEntered, onExited)

**Comment by Jojo-Schmitz** [comment]
> @RomanPudashkin I am sorry. I did not see that.

I'd bet you started working on that longer ago than the 4 hours @RomanPudashkin has that issue assigned ;-)

**Comment by RomanPudashkin** [comment]
> > @RomanPudashkin I am sorry. I did not see that.
> 
> I'd bet you started working on that longer ago than the 4 hours @RomanPudashkin has that issue assigned ;-)

I think we need some kind of instruction to prevent such situations. Probably someone who wants to take an issue should write something in the comments of the issue (something like "I will take this issue"). Thus, the next developer who opens this issue will notice the comment and will not take this issue for himself :)

@shoogle what do you think about this? :)

**Comment by Mystic-Slice** [comment]
I will create a new pull request with all the required changes. I closed this because I could not remove the merge commit

**Comment by Jojo-Schmitz** [comment]
That should have been easy: `git pull --rebase musescore master; git push -f` (if your upstream is called 'musescore`)
Then `git rebase -i HEAD~2` and drop the nduplicate fix, again `git push -f`

**Comment by shoogle** [comment]
> I think we need some kind of instruction to prevent such situations. Probably someone who wants to take an issue should write something in the comments of the issue (something like "I will take this issue").

Yes, we should start encouraging people to do this. Initially, only team members can be assigned to issues. However, once a contributor has commented on an issue it is possible for team members to properly assign them (see https://github.com/musescore/MuseScore/issues/7689#issuecomment-791966420 for example).

I'd like to have a bot do this automatically when people say "please assign me" but that takes time to set up obviously. I'll ask in Discord to see if somebody feels like making one for us.

**Pull request closed without merging**